src/controller/python/.gitignore

```diff
@@ -0,0 +1,2 @@
+_ChipDeviceCtrl.so
```

Comment:
Why? Artifacts should be in the output dir.

Reply:
removed, yes, it is in output dir. thanks